packages/react-components/react-avatar/src/components/Avatar/useAvatar.tsx

```diff
@@ -44,6 +44,22 @@ export const useAvatar_unstable = (props: AvatarProps, ref: React.Ref<HTMLElemen
     /* excludedPropNames: */ ['name'],
   );
 
+  const [imageHidden, setImageHidden] = React.useState<true | undefined>(undefined);
```

Comment:
nit: Can you please clarify why it can't be simply `boolean` with `false` initially?

```suggestion
  const [imageHidden, setImageHidden] = React.useState<boolean>(false);
```

Reply:
> To avoid adding DOM attribute?

Yes, that is the reason.